node/src/reactor/validator.rs

```diff
@@ -790,6 +790,9 @@ impl reactor::Reactor for Reactor {
                             });
                         self.dispatch_event(effect_builder, rng, reactor_event)
                     }
+                    ConsensusAnnouncement::DisconnectFromPeer(_peer) => {
+                        todo!("implement disconnection")
```

Comment:
I am not sure it's a good idea to use a `todo!` here. It will panic, right?

Reply:
Maybe just a warning and leave it unhandled for now. Also, add a `TODO` comment and maybe a follow-up ticket?

Reply:
I think it will. That would be motivation to get this sorted ASAP! :grin:

But seriously, I'm not sure, either. I could just output a debug message and return here instead. I'm open to hearing what others think about it, too.

Reply:
We shouldn't be breaking the software. This other end of the feature is not implemented yet.